packages/suite/src/actions/suite/suiteActions.ts

```diff
@@ -18,7 +18,11 @@ import type {
     ButtonRequest,
     AppState,
 } from '@suite-types';
-import type { DebugModeOptions, AutodetectSettings } from '@suite-reducers/suiteReducer';
+import {
+    type DebugModeOptions,
```

Comment:
I believe there is no need to have `type` before at all

Reply:
done

Reply:
import type prevents from circular dependencies, that is why we start to using it.
as i recall we already had this discussion before (result on [slack](https://satoshilabs.slack.com/archives/G019WLX2P7B/p1643539186265879) but i cant find the exact thread) @Nodonisko even prepared some examples and as i remember it end up that we are missing something in setup (i cant remembrer exactly what) and correct me if im wrong but i think we agree that using `import type` is not such a bad idea after all

Reply:
@szymonlesisz I can't find discussion too, but long story short:

1. If you are compiling code using TS **`tsc`, compiler knows full context and will remove import statement** even without `type` anotation because compiler knows what is type and what is value. You can try and check this in `suite/libDev` output, you can even combine type and value in one import and will correctly strip only types.
2. If you are using `babel` to compile TS, like we do in webpack that building suite, babel can't tell if it's type or value from import statement. Babel always works in **only one file context**, but it can tell from usage in rest of file:
```tsx
import { colors, Color } from './colors';
// babel see here it is used as type
const getColor = (colorName: Color) => colors[colorName];
```
Babel see that `Color` is used as type and can safely remove `Color` from code even when it doesn't see context of other.

So what will happen if `babel` can't tell from usage if it's import or value? Consider this code:
```tsx
import { Color } from './colors';
export { Color };
```
In this file babel has no idea if he can remove Color or not. Luckily we have everywhere enabled TS option `isolatedModules: true,` which will throw TS error in all cases where it's not clear if it's value or type and ask you to explicitely add `import type`, so babel will know what can be removed.

These are all scenarios that can occur. In past you probably needed to add `import type` because TS was not properly configured with `isolatedModules: true` option, but that is already few months fixed.

To sum it up: **Don't use `import type` unless TS will throw error and explicitely ask you to do that.** I have nothing agains using, so if you like this more feel free to use it, I won't pointing this out anymore in PR.  Just please be aware how it works and that it has no benefits and will not prevent any circular deps.